app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -85,8 +84,9 @@ class GithubWebhook extends RequestHandler<Body> {
           }
           break;
         case 'create':
-          branchService = BranchService(datastore, rawRequest: stringRequest);
-          await branchService!.handleCreateRequest();
+          await branchService.handleCreateRequest(stringRequest);
+          final CreateEvent createEvent = _getCreateEvent(stringRequest)!;
+          await branchService.branchFlutterRecipes(createEvent.ref!);
```

Comment:
Use existing `CreateEvent` logic in branch_server and simply inject stringRequest to `branchFlutterRecipes`.

Reply:
I did the reverse, and moved the CreateEvent parse logic into github_webhook (I don't think the branch service should be dependent on the raw request)